src/collective/cover/layout.py

```diff
@@ -257,8 +256,59 @@ def transform(self, layout):
                     element['class'] = 'tile'
 
     def columns_formatter(self, columns):
-        # This formatter works for Deco; you can implement a custom one
-        # for you grid system
+        raise Exception('Must be implemented in the child')
+
+
+class Bootstrap3(BaseGrid, grok.GlobalUtility):
+
+    """Bootstrap 3 grid system (12 columns)."""
+
+    grok.name('bootstrap3')
+    grok.implements(IGridSystem)
+
+    ncolumns = 12
+    title = _(u'Bootstrap 3')
+
+    def columns_formatter(self, columns):
+        prefix = 'col-md-'
```

Comment:
I wonder if we need and/or want to add `col-xs-` as well (and maybe `col-sm` and `col-lg`), with different values. But I guess that would need extra settings on the cover: `column-size-xs`, etcetera, including a way for the user to set that in the Plone UI on the layout tab. That sounds too complex, and certainly not something that would be needed in this pull request.